### PR TITLE
fix: failures of preview-link in Node.js runtime

### PR DESCRIPTION
**Pull request by PierrickVoulet**
* Only send one response in all cases.
* Filter card click events before matched URLs to avoid unreachable code.

### CODE REVIEW

File: node/preview-link/attach-card.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ exports.onMessage = (req, res) => {
   if (req.method === 'GET' || !req.body.message) {
     res.send(
```
Comment (vinay-google) [inline node/preview-link/attach-card.js:27]:
```suggestion
    return res.send(
```

File: node/preview-link/attach-card.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ exports.onMessage = (req, res) => {
...
       'Hello! This function is meant to be used in a Google Chat Space.');
+    return;
```
Comment (vinay-google) [inline node/preview-link/attach-card.js:29]:
```suggestion
```

File: node/preview-link/attach-card.js [hunk 3]
```diff
@@ -97,6 +98,7 @@ exports.onMessage = (req, res) => {
         },
       ],
     });
+    return;
```
Comment (vinay-google) [inline node/preview-link/attach-card.js:101]:
```suggestion
```

**Comment by vinay-google** [comment]
Instead of having the `return` statement on its own line, we can just do it wherever we are either doing `res.send` or `res.json`. For example:

```javascript
return res.send('Hello! This function is meant to be used in a Google Chat Space.');
```

**Comment by PierrickVoulet** [comment]
> Instead of having the `return` statement on its own line, we can just do it wherever we are either doing `res.send` or `res.json`. For example:
> 
> ```js
> return res.send('Hello! This function is meant to be used in a Google Chat Space.');
> ```

Agreed and done!

**Review by vinay-google — APPROVED** [review]

**Pull request merged**